frontend/src/app/components/block-overview-graph/tx-view.ts

```diff
@@ -140,6 +142,16 @@ export default class TxView implements TransactionStripped {
   }
 
   getColor(): Color {
+    // Block audit
+    if (this.status === 'found') {
+      // return hexToColor('1a4987');
```

Comment:
Is this intentional?

Reply:
Yes, we're still in draft 👍🏻 . Once we settle on the color/animation style this will be cleaned up.